lib/rucio/client/uploadclient.py

```diff
@@ -62,7 +62,7 @@
 
 class UploadClient:
 
-    def __init__(self, _client=None, logger=None, tracing=True):
+    def __init__(self, _client=None, logger=logging.log, tracing=True):
```

Comment:
Check consistency see above. If you add `logger=logging.log` here, check that no-one explicitly passes `None` and remove the `if not logger:` below.